plugins/modules/rpm_sync.py

```diff
@@ -73,7 +108,20 @@ def main():
         repository = PulpRpmRepository(module, {"name": module.params["repository"]})
         repository.find(failsafe=False)
 
-        repository.process_sync(remote)
+        # pulp_rpm supports sync_policy from 3.16.
+        # Earlier versions support only mirror.
+        rpm_version = module.pulp_api.api_spec["info"]["x-pulp-app-versions"]["rpm"]
```

Comment:
Can you catch any error around this, and assume that versions not having it are definitely older than 3.16?

Reply:
How about this:

```
rpm_version = module.pulp_api.api_spec.get("info", {}).get("x-pulp-app-versions", {}).get("rpm", ())
```

Which would result in an empty tuple, and always appear as a lower version?

Reply:
sounds good.

Reply:
done